src/Data/CodePoint/Unicode/Internal.purs

```diff
@@ -1,10 +1,10 @@
 -----------------------------------------------------------
 -- This is an automatically generated file: do not edit
--- Generated by ubconfc at Fri Nov 10 20:05:16 PST 2017
+-- Generated by ubconfc at Tue Jan 12 18:57:20 CST 2021
```

Comment:
I don't have a good way to verify this file other than seeing that the tests still pass. From a scan through it looks fine to me. If you have any suggestions on verifying this, @MonoidMusician, I'm all ears!

Reply:
Also, we have some documentation that describes generating these files anew:

https://github.com/purescript-contrib/purescript-unicode/tree/main/docs#generating-internal-modules

You've made some changes to how these are generated; if there's any information you think would be helpful to maintainers working on this in the future, please consider adding it to that documentation file as well. Thanks!

Reply:
Thanks, I missed that it was moved to there, will update.